src/App.jsx

```diff
@@ -14,33 +15,89 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [activeButton, setActiveButton] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const funcSortAlphabetically = () => {
+    const sortedGoods = [...goods].sort();
```

Comment:
The sorting operation here creates a new sorted array, which is a good practice to avoid mutating the state directly.